_includes/footer_custom.html

```diff
@@ -1,3 +1,5 @@
 {%- if site.footer_content -%}
   <p class="text-small text-grey-dk-100 mb-0">{{ site.footer_content }}</p>
 {%- endif -%}
+
+  <p class="text-small text-grey-dk-100 mb-0">This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll.</p>
```

Comment:
Doesn't the indentation seem a bit off?

Reply:
Yes, my mistake. Thank you! Hopefully corrected now.